src/css/application.css

```diff
@@ -604,3 +604,21 @@ body {
 .u__hidden {
   display: none;
 }
+
+.loading {
+  align-items: center;
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  width: 100%;
+}
+
+.loading__spinner {
```

Comment:
Do we need a nested element here? Could we just have the `.loading` block fill the whole container (maybe just absolutely position it) and then put the spinner in the center with `background-position` and a `vh`/`vw` based `background-size`?